web/src/api/hooks/api.ts

```diff
@@ -24,6 +24,7 @@ type BadgerFetchOptions = {
     withCredentials?: boolean;
     plainHeaders?: boolean;
     isBlob?: boolean;
+    isText?: boolean; // todo: isBlob and isText must be changed to 1 field type
```

Comment:
I know that you want to change this in future, but my sense of perfection suffers from it today...

Reply:
I didn't refactor `isBlob` field just to skip testing almost whole project. Searching by `useBadgerFetch` returns 98 results in 21 files